codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/codegen/SymbolVisitor.java

```diff
@@ -471,6 +466,11 @@ public Symbol floatShape(FloatShape shape) {
       .build();
   }
 
+  @Override
+  public Symbol documentShape(DocumentShape shape) {
```

Comment:
Is this just for completeness or do we use this somewhere?

Reply:
Just for completeness - and not fail on document shapes.